models/experimental/functional_resnet/tt/ttnn_functional_resnet.py

```diff
@@ -100,7 +99,9 @@ def resnet_bottleneck_block(x, parameters, layer=None, module=None, device=None)
         ttnn.deallocate(identity)
 
     if (layer is not None and module is not None) and (
-        (layer == 2 and module == 1) or (layer == 1 and module == 1 and is_wormhole_b0())
+        (layer == 1 and module == 1)
+        or (layer == 1 and module == 2 and is_grayskull())
+        or (layer == 1 and module == 3 and is_grayskull())
```

Comment:
Why do we no longer need the is_wormhole_b0() ?  Should this have been kept in the condition as well?

Reply:
it just means we need that condition on both WH and GS.